docs/source/testing/mocking.md

```diff
@@ -3,13 +3,13 @@ title: Mocking
 description: Mock your GraphQL data based on a schema.
 ---
 
-The strongly-typed nature of a GraphQL API lends itself extremely well to mocking. This is an important part of a GraphQL-First development process, because it enables frontend developers to build out UI components and features without having to wait for a backend implementation.
+Mocking allows Apollo Server to return simulated data for GraphQL operations based on your server's schema. The strongly-typed nature of a GraphQL API lends itself extremely well to mocking, which is an important part of a GraphQL-first development process.  
```

Comment:
Suggest "enables" instead of "allows" (I like to limit "allows" to situations where something is granted permission)

Independent of your edits, suggest cutting "extremely well" just for brevity's sake